app/controllers/vendor_api/vendor_api_controller.rb

```diff
@@ -8,6 +8,17 @@ class VendorApiController < ActionController::API
     before_action :set_cors_headers
     before_action :require_valid_api_token!
 
+    def audit_user
+      return nil unless @metadata.present?
+
+      @audit_user ||= VendorApiUser.find_or_create_by(
+        email_address: @metadata.attribution.email,
```

Comment:
This will recreate a user if their name changes in the vendor system. Do we want to find_or_create on the basis of only `user_id` and `vendor_api_token_id`?

Reply:
Happy to change it but I'm guessing that the vendor API token can change too, though not often. Also we need to consider whether we want the information we capture to reflect the state of the user at the time or what they are now (or at least were when they last accessed the API)